--- a/moo.cabal	2018-11-12 23:10:24.000000000 +0100
+++ b/moo.cabal	2022-09-13 15:58:21.713303329 +0200
@@ -80,7 +80,6 @@
                       , Moo.GeneticAlgorithm.Selection
                       , Moo.GeneticAlgorithm.StopCondition
                       , Moo.GeneticAlgorithm.Utilities
-                      , Moo.GeneticAlgorithm.Crossover
                       , Moo.GeneticAlgorithm.Niching
 
 Test-Suite moo-tests
